src/main/java/org/kohsuke/github/GHAppCreateTokenBuilder.java

```diff
@@ -35,7 +35,7 @@
      *
      */
     @Preview @Deprecated
-    public GHAppCreateTokenBuilder repositoryIds(List<Integer> repositoryIds) {
+    public GHAppCreateTokenBuilder repositoryIds(List<Long> repositoryIds) {
```

Comment:
Does this automatically cast from `List<Integer>` to `List<Long>`. 
If not, it would be better to keep both for at least one more release (with an issue filed to remove the `List<Integer>` one). 
I know we have no commitment to maintain it, but we can still try to be nice.

Reply:
Agreed. I will deprecate this one and add the right one. Java auto-boxing won’t be a solution unfortunately. As soon as I get to work I will do it and let you know

Reply:
Nevermind!

Reply:
Since this hasn't been released a version with the `Integer` yet, we can just change it.